common/sgx/report.c

```diff
@@ -190,3 +193,56 @@ oe_result_t oe_get_target_info(
 done:
     return result;
 }
+
+oe_result_t oe_get_target_info_v2(
+    const uint8_t* report,
+    size_t report_size,
+    void** target_info_buffer,
+    size_t* target_info_size)
+{
+    oe_result_t result = OE_FAILURE;
+    size_t temp_size = 0;
+    void* temp_info = NULL;
+
+    if (!target_info_buffer || !target_info_size)
+    {
+        return OE_INVALID_PARAMETER;
+    }
+
+    *target_info_buffer = NULL;
+    *target_info_size = 0;
+
+    result = oe_get_target_info_v1(report, report_size, NULL, &temp_size);
```

Comment:
This is lame.   I thought our code did it the other way around... v2 is the clean code and v1 wraps v2.  That way when v1 is eventually deleted you don't have to rewrite the v2 code later, it's already well tested by then.

Reply:
Agreed, new_platforms does have v1 calling v2 as far as I can tell. See oe_get_target_info_v2 in sgx_common.c. I don't think all the v2 versions have been implemented in the root definitions, we might want to bring this up before they are.